admin_guide/docker_registry.adoc

```diff
@@ -14,42 +14,54 @@ toc::[]
 
 To deploy the integrated OpenShift
 link:../architecture/infrastructure_components/image_registry.html[Docker
-Registry], run the `oadm registry` command as a user with cluster
-administrator privileges:
+registry], run the following command as a user with cluster administrator
+privileges:
 
+====
 ----
-$ oadm registry --config=admin.kubeconfig --credentials=openshift-registry.kubeconfig
+$ oadm registry --config=admin.kubeconfig \
+    --credentials=openshift-registry.kubeconfig
 ----
+<1> `--config` is the path to the *_kubeconfig_* file for the cluster
+administrator.
+<2> `--credentials` is the path to the *_kubeconfig_* file for the
+*openshift-registry*.
+====
 
-where the value passed to `--config` is the path to the kubeconfig file for the
-cluster administrator, and `--credentials` is the path to the kubeconfig file
-for the openshift-registry.
+This creates a service called *docker-registry* and a deployment configuration
+called *docker-registry*. Once deployed successfully, a pod is created with a
+name similar to *docker-registry-1-cpty9*.
```

Comment:
Can we factor "*docker-registry*" out?  How about "This creates a service and a deployment configuration, both called *docker-registry*."?  Not a big deal, anyway.